### PR TITLE
Adding include-fragment-element.css to include with package

### PR DESCRIPTION
**Pull request by jonrohan**
I wanted to include the `include-fragment-element.css` with this package. The css exists to normalize the custom html elements.

Then in app we're going to include this css with

```scss
@import "include-fragment-element/include-fragment-element";
```

to: @github/js 
cc: @github/design-systems

### CODE REVIEW

File: include-fragment-element.css [hunk 1]
```diff
@@ -0,0 +1,5 @@
+include-fragment,
+poll-include-fragment {
```
Comment (josh) [inline include-fragment-element.css]: `poll-include-fragment` isn't defined by this library.

**Comment by jonrohan** [comment]
> I do agree with the idea that <include-fragment> should be treated as a block element by default, this will likely only ever be the only style included here.

For us it's more a question of organization rather than how much css we have to write. Because this was an outside library we wanted to include it from the package rather than trying to organize it internally.

File: include-fragment-element.css (hunk 1 above)
Comment (jonrohan) [inline include-fragment-element.css]: Removed it

**Comment by mislav** [comment]
I agree with @josh, but for the sake of consistency and organization, I can go with this change. 👍

**Comment by josh** [comment]
I'll get a 2.0.1 release out soon.

**Pull request merged**